README.md

```diff
@@ -41,20 +41,20 @@ _The full proof and the proving time will be displayed in the browser console._
 | Circuit                     | Constraints | zKey size |
 | --------------------------- | ----------- | --------- |
 | ecdsa_verify                | 163,239     | 119MB     |
-| ecdsa_verify_pubkey_to_addr | 466,599     | 291MB     |
+| ecdsa_verify_pubkey_to_addr | 315,175     | 197MB     |
 
 ### Browser proving
```

Comment:
I would add a disclaimer here saying this wasn't a rigorous benchmark and we are hoping to do that soon across many devices?